### PR TITLE
Added to Localization

### PR DESCRIPTION
**Pull request by mbiesiad**
Hi. First of all, great project & idea! Here: https://github.com/mbiesiad/awesome-open-source-supporters/blob/master/README.md
is Localeapp & Weblate added to Localization.

Best greetings

### CODE REVIEW

File: README.md
Comment (zachflower) [inline README.md]: I took a quick look at Weblate's pricing page and it looks like you have to contact them for Open Source pricing. Is there a review process, or is it based off of something like a GitHub link (where a public repo is automatically included in the Open Source tier)?

If approval is necessary, can we get the `requires-approval` tag added?

```suggestion
- [Weblate](https://weblate.org/) `requires-approval` - Web-based continuous localization.
```

**Review by zachflower — CHANGES_REQUESTED** [review]
Thanks for these! Only one question, otherwise ready to merge.

File: README.md
Comment (mbiesiad) [reply README.md]: Good question, so I'm participating there in several (over ~70) OSS projects, like phpMyAdmin, Debian Handbook, Coala (Py), GNU, Git Manpages. I didn't had to pay for anything (it's free).
I will compared this to the Crowdin, if I'm not mistaken - review process and/or without, depends of maintainer' preferences/settings.
The summary is, yeah, I think we can add the tag:-)

**Comment by mbiesiad** [comment]
> 
> 
> Thanks for these! Only one question, otherwise ready to merge.

My pleasure! That was a good question 🔥

**Review by zachflower — APPROVED** [review]
LGTM!

**Pull request merged**
